numpy/distutils/command/egg_info.py

```diff
@@ -1,9 +1,17 @@
 from __future__ import division, absolute_import, print_function
 
+import sys
+
 from setuptools.command.egg_info import egg_info as _egg_info
 
 class egg_info(_egg_info):
     def run(self):
+        if 'sdist' in sys.argv:
+            import warnings
+            warnings.warn("`build_src` is being run, this may lead to missing "
```

Comment:
@charris @rgommers I ran into this issue with one of the skimage tools, which checks to see whether our MANIFEST file is complete.  It says to look at gh-7127, but I'm not sure what to look at!  Could we point to a likely fix in this message instead?  I am happy to make the patch, but was hoping you could point me in the right direction.